### DISCUSSION TITLE
custom line separator for SQL editor

### DISCUSSION THREAD

**Discussion by fjf2002** [dbeaver/dbeaver#13826] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I would like to configure the SQL editor line separator to LF, even when using DBeaver on Micro$oft Windoze.

**Describe the solution you'd like**
Line sepatator could be configurable through Preferences.

**Describe alternatives you've considered**
Both `SET JAVA_TOOL_OPTIONS="-Dline.separator=\n"` or adding such line into dbeaver.ini fail, because \n is interpreted literally instead of being converted to LF.

**Additional context**
The main reason for my request is: When CREATing FUNCTIONs in PostgreSQL with the DBeaver SQL Editor, the function body will contain CR characters (when using DBeaver on Windoze). I would like to get rid of the CR chars in the FUNCTION bodies.

**Comment by ShadelessFox** [comment, marked as answer]
Hello @fjf2002,

You can configure line delimiter in preferences:

![java_ixTA2lvMuz](https://user-images.githubusercontent.com/35821147/132355974-44fc11ee-66d8-4c6d-a2d5-9fbe36ae919d.png)

**Reply by fjf2002** [reply]
Thanks, I missed that. I was searching in the editor preferences.
